spring-cloud-data-rest/src/main/java/org/springframework/cloud/data/AdminApplication.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.springframework.cloud.data.rest;
```

Comment:
Almost, but I wasn't thinking of creating another project. Rather we could rename `spring-cloud-data-rest` to `spring-cloud-data-admin`. Then the top-level package for all code in the project would be: `org.springframework.cloud.data.admin`. The following sub-packages would still exist: config, controller, repository. Only `AdminAppplication` would reside directly under the `.admin` top level package. The `UiController` would sit alongside the "REST" Controllers within the `controller` subpackage (REST being considered an implementation detail of all the other Controllers).